### PR TITLE
compat with pandas 0.20.0 for import of hashing

### PR DESCRIPTION
**Pull request by jreback**
closes # 2245

### CODE REVIEW

File: dask/dataframe/hashing.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
 # need to use this hacky awfulness to check if we're > 0.19.2.
 if PANDAS_VERSION not in ('0.19.1', '0.19.2') and PANDAS_VERSION > '0.19.0+460':
     from pandas.tools.hashing import hash_pandas_object
+elif PANDAS_VERSION >= '0.20.0':
+    from pandas.util.hashing import hash_pandas_object
```
Comment (jcrist) [inline dask/dataframe/hashing.py]: This should go above the first check, as `0.20.0 > 0.19.0+460`.

Comment (jreback) [inline dask/dataframe/hashing.py]: done

**Comment by jreback** [comment]
so we are going to have an rc1 next day or 2 (though of course it will fail *this* check)......

**Comment by jcrist** [comment]
Thanks!

**Comment by mrocklin** [comment]
Thank you for being proactive here @jreback

**Pull request merged**